src/Compilers/CSharp/Portable/Binder/Binder_Attributes.cs

```diff
@@ -102,7 +103,9 @@ internal CSharpAttributeData GetAttribute(AttributeSyntax node, NamedTypeSymbol
 
         internal BoundAttribute BindAttribute(AttributeSyntax node, NamedTypeSymbol attributeType, DiagnosticBag diagnostics)
         {
-            return this.GetBinder(node).BindAttributeCore(node, attributeType, diagnostics);
+            var binder = this.GetBinder(node);
```

Comment:
Are we at the point that we should consider adding a `GetRequiredBinder` method?

Reply:
If not we soon will be. Will add.

---
In reply to: [367115570](https://github.com/dotnet/roslyn/pull/40946#discussion_r367115570) [](ancestors = 367115570)